### PR TITLE
Define integration tests for audio (and general improvements)

### PR DESCRIPTION
**Pull request by dhruvkb**
## Dependencies
Please merge #108 before this.

## Description
This PR adds integration tests for audio, while making general code quality improvements in the areas of code duplication and organisation.

This is not a comprehensive set of tests for all audio functionality, but these tests help to establish the basic working of audio endpoints.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
Really wonderful work. ran the api and tests locally without issue.

File: openverse-api/test/media_integration.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+"""
+End-to-end API tests for audio. Can be used to verify a live deployment is
```
Comment (krysal) [inline openverse-api/test/media_integration.py:2]: Little typo here: `audio` --> `all media types` ?

File: openverse-api/test/run_test.sh [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+
+red="\e[31m"
+green="\e[32m"
+endcol="\e[0m"
+
+# Specify a file as the first argument to restrict the test to that file only.
+# ```
+# $	./test/run_test.sh test/audio_integration_test.py
+# ```
+TEST_ARG="${1:-test/}"
```
Comment (krysal) [inline openverse-api/test/run_test.sh:11]: This will be very helpful, fantastic!

**Review by krysal — APPROVED** [review]
Clean code separation as always 👍

File: openverse-api/test/media_integration.py (hunk 1 above)
Comment (zackkrida) [reply openverse-api/test/media_integration.py:2]: @dhruvkb let's make sure we get this one fixed separately

Comment (dhruvkb) [reply openverse-api/test/media_integration.py:2]: Shouldn't have merged from a phone. It didn't show any comments except "approved". Fixed in #177.

**Pull request merged**